backend/python/tapa/tapac.py

```diff
@@ -5,6 +5,7 @@
 import logging
 import os
 import os.path
+import pkg_resources
```

Comment:
`pkg_resources` is a third-party package and should be grouped with other third-party packages.